packages/serverless-contracts/src/contracts/apiGateway/types/lambdaHandler.ts

```diff
@@ -1,17 +1,21 @@
+import { APIGatewayProxyWithCognitoAuthorizerEvent } from 'aws-lambda';
```

Comment:
```suggestion
import type { APIGatewayProxyWithCognitoAuthorizerEvent } from 'aws-lambda';
```
This should allow to skip `aws-lambda` as a dependency